File: java/darts/src/main/java/Darts.java
class Darts {

	double x;
	double y;
    Darts(double x, double y) {
    	this.x = x;
    	this.y = y;
    }

    int score() {
    	double distance = Math.sqrt(this.x * this.x + this.y * this.y);
    	if (distance <= 1.0) {
    		return 10;
    	}
    	else if (distance <= 5.0) { 
    		return 5;
    	}
    	else if (distance <= 10.0) {
    		return 1;
    	}
    	else {
    		return 0;
    	}
    }

}

File: python/darts/darts.py
import math

def score(x, y):
    """
    Score a throw in a game of darts based on the distance from the center (0, 0)
    x: x-coordinate of the throw destination
    y: y-coordinate of the throw destination
    """
    distance = math.sqrt(x * x + y * y)
    if distance <= 1.0:
        return 10
    
    if distance <= 5.0:
        return 5
    
    if distance <= 10.0:
        return 1
    
    return 0
